pkg/certificate/mrc_reconciler.go

```diff
@@ -170,10 +180,42 @@ func (m *Manager) updateMRCState(mrc *v1alpha2.MeshRootCertificate) error {
 		mrc.Status.TransitionAfter = nil
 	} else {
 		mrc.Status.TransitionAfter = &metav1.Time{Time: time.Now().Add(mrcDurationPerStage)}
+		log.Debug().Str("mrc", namespacedMRCName(mrc)).Msgf("updated MRC TransitionAfter to %s", mrc.Status.TransitionAfter)
+	}
+
+	if shouldRetrieveCA(mrc) {
+		_, _, err := m.mrcClient.GetCertIssuerForMRC(mrc)
+		if err != nil {
+			log.Error().Err(err).Str("mrc", namespacedMRCName(mrc)).Msg("failed to retrieve CA on passive MRC creation")
+			if !m.leaderMode {
```

Comment:
Only the leader should attempt to retrieve the CA. There's not necessarily a reason for each control plane component to try and retrieve the CA (but I'm willing to be convinced)

Reply:
Updated so that only the leader will attempt to retrieve the CA.